### PR TITLE
fix(code): copying interpolated variables

### PR DESCRIPTION
**Pull request by kellyjosephprice**
### 🧰  Changes

The Code component uses the syntax-highlighter for variable
interpolation, but it was passing the uninterpolated text to the copy
function.

### CODE REVIEW

**Review by kunal-pathak — COMMENTED** [review]
@kellyjosephprice this looks pretty solid to me and like a positive change, but I don't have a ton of context in this area yet, so I'm gonna add @rafegoldberg in addition to myself!

File: components/Code/index.jsx [hunk 1]
```diff
@@ -8,11 +8,13 @@ const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/synta
 const { canonical: canonicalLanguage } = require('@readme/syntax-highlighter');
 const copy = require('copy-to-clipboard');
 
-function CopyCode({ code, rootClass = 'rdmd-code-copy', className = '' }) {
+function CopyCode({ codeRef, rootClass = 'rdmd-code-copy', className = '' }) {
   const copyClass = `${rootClass}_copied`;
   const button = React.createRef();
   /* istanbul ignore next */
   const copier = () => {
+    const code = codeRef.current.textContent;
```
Comment (rafegoldberg) [inline components/Code/index.jsx:16]: Slick.

File: __mocks__/copy-to-clipboard.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const copy = jest.fn(() => true);
+
+module.exports = copy;
```
Comment (rafegoldberg) [inline __mocks__/copy-to-clipboard.js:3]: lol jest...

**Review by rafegoldberg — APPROVED** [review]
👍 LGTM. Did you test this linked to ReadMe as well, or just against your unit tests?

File: components/Code/index.jsx (hunk 1 above)
Comment (kellyjosephprice) [reply components/Code/index.jsx:16]: Let me double check. I tested before writing the tests and stuff.

Comment (kellyjosephprice) [reply components/Code/index.jsx:16]: Ok, after much linkage, it works for me locally.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.22.3</b>](/readmeio/markdown/releases/tag/v6.22.3)
<br>

**Pull request merged**